insights/parsers/bond.py

```diff
@@ -38,22 +38,22 @@
 ``/proc/net/bonding``.
 
 Examples:
-    >>> bond_info = shared[Bond]
+    >>> type(bond_info)
+    <class 'insights.parsers.bond.Bond'>
     >>> bond_info.bond_mode
-    '0'
+    '4'
```

Comment:
The test result of the examples better match with above typical output.

Reply:
@xiangce - Bond mode is 4, it's not active-backup(mode-0)